server/src/main/java/com/unosquare/carmigo/entity/PlatformUser.java

```diff
@@ -63,6 +64,7 @@
     @Exclude
     @ManyToOne(cascade = CascadeType.DETACH, fetch = FetchType.LAZY)
     @JoinColumn(name = "user_access_status_id", nullable = false)
+    @JsonIgnoreProperties({"hibernateLazyInitializer"})
```

Comment:
Hi Kleyton! 

I am curious as to why we need this property on the entity?  Is an object of this type being serialised somewhere?

Reply:
When hibernate does a Lazy initialisation process it adds this field (hibernateLazyInitializer) to the PlatformUser object. When converting the values from PlatformUser to JsonNode, ObjectMapper doesn't know what to do with this extra field (because of UserAccessStatus) and then blows up.

https://stackoverflow.com/questions/67353793/what-does-jsonignorepropertieshibernatelazyinitializer-handler-do
Taking a look into the object, I see hibernateLazyInitializer field but not the handler field as described in stackoverflow. That is why I tell Json to ignore this field only.

Reply:
Okay, It might be an idea to use modelmapper to map from PlatformUser entity to PlatformUserDTO and then convert that to JsonNode?  It follows the pattern of Entity to DTO for further processing that is used elsewhere?

Reply:
It has worked!